project-base/scripts/install-docker-wsl-debian.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+GREEN="\e[32m"
+NC="\e[0m"
+
+echo This is installation script that will install Docker in your Debian on Windows. You will be prompted to enter your password in order to install all necessary applications.
+
+printf "${GREEN}Updating and installing necessary applications${NC}\n"
+
+sudo apt update
+sudo apt install -y --no-install-recommends apt-transport-https ca-certificates curl gnupg2 software-properties-common wget lsb-release apt-transport-https
+sudo apt-get update -y && sudo apt-get install -y apt-transport-https ca-certificates curl gnupg-agent software-properties-common wget lsb-release apt-transport-https
```

Comment:
Is it necessary to install some apps twice (apt-transport-https, ca-certificates, curl, software-properties-common, wget, lsb-release, apt-transport-https)?